src/test/java/specs/FixturesSpec.java

```diff
@@ -486,7 +490,7 @@
           it("report each error", () -> {
             assertThat(result.get().getFailureCount(), is(2));
 
-            assertThat(failureMessages.get(), hasItem("boom 1"));
+            assertThat(failureMessages.get(), hasItem("java.lang.Exception: boom 1"));
```

Comment:
The checked exception gets wrapped. This only happens in the innermost consumer. Outer layers are (and can be) more permissive of unchecked exceptions.